src/modules/playRoom/components/ChatBoxComponents/Message.js

```diff
@@ -0,0 +1,28 @@
+import { connect } from "react-redux";
+import UserMessage from "./UserMessage";
+import Notification from "./Notification";
+
+const Message = (props) => {
+  if (!props.message || !props.message.type) {
```

Comment:
Why you have to check message and message type?

Reply:
checking message and message.type is undefined or not as per our teacher instruction.

Reply:
Use module ```prop-types``` for this